### PR TITLE
SSO Login buttons component added with Microsoft Login and existing logins

### PR DESCRIPTION
**Pull request by kandaj**

### CODE REVIEW

**Review by RobinKavPA — APPROVED** [review]

**Review by RobinKavPA — APPROVED** [review]

File: src/pages/commonComponents/LoginSSOButtons/LoginSSOButttons.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<!--- Checkbox.stories.mdx -->
```
[the remark this answers is not in the document]
Comment (kandaj) [reply src/pages/commonComponents/LoginSSOButtons/LoginSSOButttons.stories.mdx]: done

File: src/pages/commonComponents/LoginSSOButtons/LoginSSOButtons.jsx [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+										) : null}
+									</button>
+								</div>
+							</Col>
+						))}
+						<Col sm={0} lg={1} key={'col-1-' + index} />
+					</Row>
+				</Fragment>
+			))}
+			<div style={{ fontSize: '12px' }}>
```
[the remark this answers is not in the document]
Comment (kandaj) [reply src/pages/commonComponents/LoginSSOButtons/LoginSSOButtons.jsx]: done

File: src/pages/commonComponents/LoginSSOButtons/LoginSSOButtons.jsx [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+	}, [lastChoice]);
+
+	return (
+		<div>
+			{_.map(_.chunk(ssoBtnsState, 2), (arr, index) => (
+				<Fragment key={index}>
+					<br />
+					<Row className='mt-2'>
+						<Col sm={0} lg={1} />
+						{_.map(arr, (value, i) => (
```
[the remark this answers is not in the document]
Comment (kandaj) [reply src/pages/commonComponents/LoginSSOButtons/LoginSSOButtons.jsx]: done

File: src/pages/commonComponents/LoginSSOButtons/LoginSSOButtons.jsx [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+								data-testid='communityLink'
+								rel='noopener noreferrer'>
+								Suggest another Indentity Provider
+							</a>
+						</span>
+					</Col>
+					<Col sm={6} lg={5}>
+						{lastChoice ? (
+							<span>
+								<img src={lastChoiceSVG} width='20' data-testid='lastChoiceNote' alt='lastChoice' /> Last Time you clicked this button
```
[the remark this answers is not in the document]
Comment (kandaj) [reply src/pages/commonComponents/LoginSSOButtons/LoginSSOButtons.jsx]: done

File: src/pages/commonComponents/LoginSSOButtons/LoginSSOButtons.jsx [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+								<div className='gray800-14'>
+									<button
+										data-testid={value.id}
+										className='btn btn-outline-secondary btn-block'
+										style={{ textAlign: 'left' }}
+										onClick={() => clickHandler(value.id, value.authURL)}>
+										<img src={value.img} width='20' style={{ float: 'left' }} alt={value.id} />
+										&nbsp; {value.text}
+										{value.active === true ? (
+											<img
```
[the remark this answers is not in the document]
Comment (kandaj) [reply src/pages/commonComponents/LoginSSOButtons/LoginSSOButtons.jsx]: done

File: src/pages/commonComponents/LoginSSOButtons/LoginSSOButtons.jsx [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+						<Col sm={0} lg={1} />
+						{_.map(arr, (value, i) => (
+							<Col sm={6} lg={5} key={value.id + i} className='mt-1'>
+								<div className='gray800-14'>
+									<button
+										data-testid={value.id}
+										className='btn btn-outline-secondary btn-block'
+										style={{ textAlign: 'left' }}
+										onClick={() => clickHandler(value.id, value.authURL)}>
+										<img src={value.img} width='20' style={{ float: 'left' }} alt={value.id} />
```
[the remark this answers is not in the document]
Comment (kandaj) [reply src/pages/commonComponents/LoginSSOButtons/LoginSSOButtons.jsx]: done

File: src/pages/commonComponents/LoginSSOButtons/LoginSSOButtons.jsx [hunk 7]
```diff
@@ -0,0 +1,97 @@
...
+										&nbsp; {value.text}
+										{value.active === true ? (
```
[the remark this answers is not in the document]
Comment (kandaj) [reply src/pages/commonComponents/LoginSSOButtons/LoginSSOButtons.jsx]: done

File: src/pages/commonComponents/LoginSSOButtons/LoginSSOButttons.stories.mdx [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+
+import { Meta, Story, Canvas } from '@storybook/addon-docs';
```
[the remark this answers is not in the document]
Comment (kandaj) [reply src/pages/commonComponents/LoginSSOButtons/LoginSSOButttons.stories.mdx:1]: not able to resolve this bootstrap overriding the font family

**Review by ThinkingMonkeyLtd — APPROVED** [review]

**Review by PaulMcCaffertyPA — APPROVED** [review]

**Pull request merged**